app/views/home/index.html.erb

```diff
@@ -1,3 +1,16 @@
 <p style="font-size: 22px;">
   Первый в мире удобный менеджер флеш-карточек. Именно так.
 </p>
+<% if flash[:notice] %>
+  <%= flash[:notice] %>
+<% end %>
+<p> Оригинальный текст карты: </p>
+<% @random_card.to_a.each do |random_card| %>
```

Comment:
why you use .to_a? and maybe will be better if random_card will contain only one card?